plugins/base/src/main/kotlin/transformers/pages/comments/DocTagToContentConverter.kt

```diff
@@ -39,14 +39,14 @@ open class DocTagToContentConverter : CommentsToContentConverter {
                 )
             )
 
-        fun buildList(ordered: Boolean, start: Int = 1) =
+        fun buildList(ordered: Boolean, newStyles: Set<Style> = emptySet(), start: Int = 1) =
             listOf(
                 ContentList(
                     buildChildren(docTag),
                     ordered,
                     dci,
                     sourceSets.toDisplaySourceSets(),
-                    styles,
+                    if (newStyles.isEmpty()) styles else styles + newStyles,
```

Comment:
Why!? It is not essential optimization.

Reply:
Anyway this  is not a good place for optimization. It is an issue of implementation of `plus`. 

This make code more complex to read and can consider as boilerplate.